src/odemis/driver/simcam.py

```diff
@@ -142,6 +143,48 @@ def __init__(self, name, role, image, dependencies=None, daemon=None, blur_facto
         # Convenience event for the user to connect and fire
         self.softwareTrigger = model.Event()
 
+        # show_message(None, 'starting and error wise thread', 'test error', timeout = 1.0)
+        self._errorCreationThread = threading.Thread(target = self._state_error_creation,
+                                                     args = (),
+                                                     name = "Creating and state error")
+        self._errorCreationThread.daemon = True
+        self._errorCreationThread.start()
+
+    def _state_error_creation(self):
+        '''
+        by creating the files 'simcam.HW.error' or 'simcam.HW.good' in the folder '/var/run/odemisd' and hardware
+        error can be produced in the Sim Cam or the error can be 'fixed'
+        After the error is produced the files should be deleted
+        :return:
+        '''
+        def attempt_to_delete_file(del_file):
+            try:
+                os.remove(del_file)
+            except:
+                pass
+
+        path = '/var/run/odemisd'
+        [attempt_to_delete_file(del_file) for del_file in [path + '/simcam.HW.error', path + '/simcam.HW.good']]
```

Comment:
Please put the filename in a constant (defined at the top of the file). Eg:
`ERROR_STATE_FILE = "simcam-hw-error"`
Also, note I'd suggest a different name. In particular, it's best to have at most one . in the filename, for the extension. As this file contains *nothing*, it's fine to have not extension.

Reply:
changed it to one dot file extension is now .error or .good